osgp/protocol-adapter-dlms/osgp-jasper-interface/src/main/java/org/opensmartgridplatform/adapter/protocol/jasper/config/JasperWirelessConfig.java

```diff
@@ -137,6 +140,11 @@ public int jasperGetSessionRetries() {
     return Integer.parseInt(this.retries);
   }
 
+  @Bean
+  public short jasperGetValidityPeriod() {
+    return Short.parseShort(this.validityPeriod);
```

Comment:
Seems unnecessary when the type is changed to `short`

Reply:
I did this because this way on a other bean this is done the same......